src/wagtail_ai/static_src/utils.tsx

```diff
@@ -19,7 +35,9 @@ const fetchAIResponse = async (
   formData.append('text', text);
   formData.append('prompt', prompt.uuid);
   try {
-    const res = await fetch(window.WAGTAIL_AI_PROCESS_URL, {
+    const aiProcessUrl = getAIConfiguration()?.wagtailAiProcessUrl;
```

Comment:
**Suggestion:** There's no need for `?` given GetAIConfiguration always returns a value thanks to its type.

Reply:
Removed in https://github.com/wagtail/wagtail-ai/commit/17298857b64c6fa1c72f307764522751b3b81fac.